intro/models.md

```diff
@@ -151,21 +151,24 @@ As you can see in the example, due to the different column types, the values can
+| `url`                                          | `string`                                                                               | `https://seatable.io`                                                                                                                                                 |                                                                                                                                                         |
 | `checkbox`                                     | `boolean`                                                                              | `true`                                                                                                                                                                |                                                                                                                                                         |
-| `rate`                                         | `integer`                                                                              | 4                                                                                                                                                                     |
+| `rate`                                         | `integer`                                                                              | 4                                                                                                                                                                     |                                                                                                                                                         |
 | `geolocation`                                  | `object`                                                                               | `{"lng": 8.23..., "lat": 50.00...}` or<br/>`{"country_region": "Sweden"}`                                                                                             | Depending on column setting.                                                                                                                            |
 | `formula`                                      | `string`                                                                               | `"Albert Summer"`                                                                                                                                                     | Contains only the result of the formula.                                                                                                                |
```

Comment:
The `/api-gateway` endpoints can also return `integer` or `boolean` values for `formula` columns.

The `/dtable-server` endpoints _seem_ to return strings in all cases.